data/building-hours/9-osa.yaml

```diff
@@ -0,0 +1,17 @@
+name: OSA
+subtitle: Office of Student Activities
```

Comment:
The only one left is HOM. 

Could you switch it to Name+subtitle as well, please?